src/librustc/session/config.rs

```diff
@@ -792,6 +794,8 @@ pub fn rustc_short_optgroups() -> Vec<RustcOptGroup> {
         opt::multi("A", "allow", "Set lint allowed", "OPT"),
         opt::multi("D", "deny", "Set lint denied", "OPT"),
         opt::multi("F", "forbid", "Set lint forbidden", "OPT"),
+        opt::multi("", "cap-lints", "Prevent lint levels from being \
+                                     more restrictive", "LEVEL"),
```

Comment:
This message is not very user-friendly, perhaps "Set the most restrictive lint level. More restrictive lints are capped at this level". Or something more concise.